.tekton/operator-bundle-pipeline.yaml

```diff
@@ -660,3 +663,129 @@ spec:
+            "revision": "$(tasks.wait-for-operator-image.results.GIT_REF)"
+          },
+          {
+            "name": "operator-bundle",
+            "containerImage": "$(tasks.build-container.results.IMAGE_URL)@$(tasks.build-container.results.IMAGE_DIGEST)",
+            "repository": "$(params.git-url)",
+            "revision": "$(params.revision)"
+          },
+          {
+            "name": "roxctl",
+            "containerImage": "$(params.roxctl-image-build-repo)@$(tasks.wait-for-roxctl-image.results.IMAGE_DIGEST)",
+            "repository": "$(tasks.wait-for-roxctl-image.results.GIT_REPO)",
+            "revision": "$(tasks.wait-for-roxctl-image.results.GIT_REF)"
+          },
+          {
+            "name": "scanner-db",
+            "containerImage": "$(params.scanner-db-image-build-repo)@$(tasks.wait-for-scanner-db-image.results.IMAGE_DIGEST)",
+            "repository": "$(tasks.wait-for-scanner-db-image.results.GIT_REPO)",
+            "revision": "$(tasks.wait-for-scanner-db-image.results.GIT_REF)"
+          },
+          {
+            "name": "scanner-db-slim",
+            "containerImage": "$(params.scanner-db-slim-image-build-repo)@$(tasks.wait-for-scanner-db-slim-image.results.IMAGE_DIGEST)",
+            "repository": "$(tasks.wait-for-scanner-db-slim-image.results.GIT_REPO)",
+            "revision": "$(tasks.wait-for-scanner-db-slim-image.results.GIT_REF)"
+          },
+          {
+            "name": "scanner",
+            "containerImage": "$(params.scanner-image-build-repo)@$(tasks.wait-for-scanner-image.results.IMAGE_DIGEST)",
+            "repository": "$(tasks.wait-for-scanner-image.results.GIT_REPO)",
+            "revision": "$(tasks.wait-for-scanner-image.results.GIT_REF)"
+          },
+          {
+            "name": "scanner-slim",
+            "containerImage": "$(params.scanner-slim-image-build-repo)@$(tasks.wait-for-scanner-slim-image.results.IMAGE_DIGEST)",
+            "repository": "$(tasks.wait-for-scanner-slim-image.results.GIT_REPO)",
+            "revision": "$(tasks.wait-for-scanner-slim-image.results.GIT_REF)"
+          },
+          {
+            "name": "scanner-v4-db",
+            "containerImage": "$(params.scanner-v4-db-image-build-repo)@$(tasks.wait-for-scanner-v4-db-image.results.IMAGE_DIGEST)",
+            "repository": "$(tasks.wait-for-scanner-v4-db-image.results.GIT_REPO)",
+            "revision": "$(tasks.wait-for-scanner-v4-db-image.results.GIT_REF)"
+          },
+          {
+            "name": "scanner-v4",
+            "containerImage": "$(params.scanner-v4-image-build-repo)@$(tasks.wait-for-scanner-v4-image.results.IMAGE_DIGEST)",
+            "repository": "$(tasks.wait-for-scanner-v4-image.results.GIT_REPO)",
+            "revision": "$(tasks.wait-for-scanner-v4-image.results.GIT_REF)"
+          }
+        ]
+    - name: PRODUCT_VERSION
+      value: $(tasks.determine-main-image-tag.results.IMAGE_TAG)
```

Comment:
Do you mind moving `PRODUCT_VERSION` before `COMPONENTS`?
When there's a large argument like `COMPONENTS`, I'm used to find smaller ones to be provided before it so that when reading the code I can pick up quickly what's the idea of the rest-of parameters without having to scroll to the end. Not sure I could explain it well but if you do the same, you could relate.